src/components/Login.jsx

```diff
@@ -12,8 +12,8 @@ const Login = (props) => {
             <h1>Login</h1>
             <section id="login">
                 <form>
-                    <Input id={'username'} onLoginChanges={props.onLoginChange} loginDetails={props.loginDetails}/>
-                    <Input id={'password'} onLoginChanges={props.onLoginChange} loginDetails={props.loginDetails}/>
+                    <Input id={'username'} onChange={props.onLoginChange} inputVal={props.loginDetails.username}/>
+                    <Input id={'password'} onChange={props.onLoginChange} inputVal={props.loginDetails.password}/>
```

Comment:
```suggestion
                    <Input id="username" onChange={props.onLoginChange} inputVal={props.loginDetails.username}/>
                    <Input id="password" onChange={props.onLoginChange} inputVal={props.loginDetails.password}/>
```

Reply:
You can also handle the login form state change in this component instead of passing in another property to this component. You can then pass in a prop that handles the submission on the login button click. That function can consume the form state managed on this component level.